internal/command/fmt.go

```diff
@@ -102,7 +103,10 @@ func (c *FmtCommand) Run(args []string) int {
 		buf := output.(*bytes.Buffer)
 		ok := buf.Len() == 0
 		if list {
-			io.Copy(&cli.UiWriter{Ui: c.Ui}, buf)
+			if _, err := io.Copy(&cli.UiWriter{Ui: c.Ui}, buf); err != nil {
+				// Should never happen
+				panic(err)
```

Comment:
I understand that this is not meant to happen, but if ever does, this panic is changing the currently existing behaviour.
Wouldn't be better to just log an error instead?

Or even better, since this is already in the part of the method that returns an exit code, we could return a different exit code that we could document.

Reply:
Fixed in 5ddc3588cc